packages/contracts/test/cyberDropBase.spec.ts

```diff
@@ -953,477 +628,79 @@ describe('CyberDropBase', function () {
     })
   })
 
-  describe('getDrop', () => {
-    it('Should throw to get drop if not exist', async () => {
```

Comment:
this test must be keep, maybe you need to change the way to check if drop exist or not but it need to throw when drop doesn't exist